bin/pgaudit_analyze

```diff
@@ -670,7 +680,7 @@ while(!$bDone)
             $strLogFile = $strNextLogFile;
             undef($strNextLogFile);
 
-            syswrite($hLog, "reading ${strLogFile}\n");
+            syswrite($hLog, "${\getTimestamp()}: reading ${strLogFile}\n");
```

Comment:
This is a good idea, but we should also do it anywhere else we write to the log file.

Reply:
ok, will do new PR shortly